### PR TITLE
Increase the maximum size for PDFs and SVGs

### PR DESCRIPTION
**Pull request by rdumusc**

### CODE REVIEW

File: dc/core/PDFContent.cpp [hunk 1]
```diff
@@ -47,6 +47,8 @@
 
 BOOST_CLASS_EXPORT_GUID( PDFContent, "PDFContent" )
 
+const qreal PDFContent::maxScale = 6;
```
Comment (tribal-tec) [inline dc/core/PDFContent.cpp]: the 'other' content maxScale can be changed from the configuration file. It seems we need/have two different scales, one for vectorial content and one for rasterized content. Both should be configurable from the configuration imo.

Comment (rdumusc) [inline dc/core/PDFContent.cpp]: OK, will do that

**Comment by rdumusc** [comment]
updated

**Comment by tribal-tec** [comment]
+1

**Pull request merged**